.github/workflows/ci.yaml

```diff
@@ -0,0 +1,68 @@
+name: CI
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      DJANGO_SETTINGS_MODULE: rca.settings.test
+      SECRET_KEY: testingsecretkey
+      SECURE_SSL_REDIRECT: false
+      ALLOWED_HOSTS: localhost
+      DATABASE_URL: postgres://postgres:postgres@localhost/postgres
+      BIRDBATH_REQUIRED: false
+
+    services:
+      postgres:
+        image: postgres:11.5
```

Comment:
@kevinhowbrook this matches the version of postgres that was used in the circle ci tests, but not the version that I can see in heroku (I only have access to staging) or indeed in `docker-compose.yml` where it is different again.

Reply:
yea it's a common (frustrating) thing that happens. I think for now match the docker compose file